tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -604,7 +604,10 @@ private void multiLevelNestedActivity(CompletionType how, int nestedCnt) throws
         assertFalse("multiLevelNestedActivity: top level LRA should be active (path called " + resourcePath.getUri() + ")",
                 lraTestService.isLRAFinished(URI.create(lraArray[0])));
 
-        lraTestService.waitForCallbacks(lra);
+        // starting at index 1 as LRAResource#multiLevelNestedActivity returns the top-level LRA as the first argument which was not finished yet
+        IntStream.range(1, uris.length).forEach(i -> {
```

Comment:
looks good, but maybe we can `.parallel()` this?

Reply:
Ok, I added the `.parallel()` here.